tests/expectations/basic-resolutions/lib/index.js

```diff
@@ -1,11 +1,18 @@
 "use strict";
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+    if (k2 === undefined) k2 = k;
+    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
+}) : (function(o, m, k, k2) {
+    if (k2 === undefined) k2 = k;
+    o[k2] = m[k];
+}));
```

Comment:
FWIW, most of the changes in these fixtures are directly called out in the blog post for 3.9:

> **Exports Now Use Getters for Live Bindings**
> When targeting module systems like CommonJS in ES5 and above, TypeScript will use get accessors to emulate live bindings so that changes to a variable in one module are witnessed in any exporting modules. This change is meant to make TypeScript’s emit more compliant with ECMAScript modules.
>